src/index.html

```diff
@@ -5,9 +5,612 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Title</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cookie&family=Rubik:wght@300;400&display=swap" rel="stylesheet">
+    <link rel="icon" href="images/logo/logo.svg">
   </head>
+
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <div class="header__menu menu">
+        <img
+          class="header__logo logo"
+          src="images/logo/logo.svg"
+          alt="logo"
+        >
+        <nav class="menu__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#bakery" class="nav__link">
+                Bakery
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#bakershops" class="nav__link">
+                Bakershops
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#about" class="nav__link">
+                About
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts" class="nav__link">
+                Contacts
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="menu__contacts">
+          <div class="menu__phone">
+            <a href="tel:+38 066 336 36 36" class="menu__phone-link">
+              +38 066 336 36 36
+            </a>
+          </div>
+          <button class="menu__button button">
+            Call back
+          </button>
+          <select
+            class="menu__language"
+            name="language"
+            id="">
```

Comment:
Не залишай пустим значення в атрибутах
```suggestion
          <select
            class="menu__language"
            name="language"
            id=""
          >
```